.github/workflows/test.yml

```diff
@@ -2,13 +2,13 @@ name: ovn-ci
 
 on:
   push:
-  pull_request:
+  pull_request_target:
```

Comment:
we need to use pull_request, 
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target
> This event runs in the context of the base of the pull request, rather than in the merge commit as the pull_request event does.
..
> Warning: The pull_request_target event is granted a read/write repository token and can access secrets, even when it is triggered from a fork

it seems you need to create a secret for this new action mikepenz/action-junit-report@v2

Reply:
Well, it's not critical; just wanted to have my junit results in a nicer format. I'll look on to how to make it work.

Reply:
I would love to have a better reporting too